### ISSUE TITLE
temporarily download from an instance provided by a url parameter

### ISSUE THREAD

**Issue by KwiatekMiki** [imputnet/cobalt#813]
### describe the feature you'd like to see

for example, when you would go to https://cobalt.tools?i=https://api.instance.example, it would download from `api.instance.example`.

this would be useful since hosting your frontend is not needed anymore and it's harder than just copying the example docker-compose file and now i only see 6 people (only counting safe instances from hyper's instance list) hosting it for the new cobalt.

<br>

thanks to that, let's say youtube downloading isn't working on the main instance but it works on an instance you found on an instance list, for example `api.instance.example`, and you just want to download from it once.

before, (most of the times) you clicked the frontend link, https://instance.example, and pasted the link.

now, you need to open the normal cobalt, open settings, instances, enable "use a custom processing server", paste the link, wait 15 seconds to be able to continue, click "save", paste the link, go back into settings and disable "use a custom processing server".

### additional context

if you try to set an instance in the settings for the first time, you get this message:
![image](https://github.com/user-attachments/assets/b0bd8fa1-4262-4821-b762-f5fd3f11c2f4)

i feel like it should also show that message but only once until you clear your cookies (like in the settings) or once per instance 

---

i hope you understand what i'm trying to say 

tl;dr: https://cobalt.tools?i=https://api.instance.example would be faster than going into settings to set the instance and later disable it, and i think it should show the important safety notice once ever/once per instance
